app/presenters/migration/new.rb

```diff
@@ -15,6 +15,112 @@ def page_content_heading
       'Tell your story'
     end
 
+    def fields
+      {
+        items: [
+          {
+            is_required: true,
+            label:      'Media',
+            name:       'ore_aggregation[edm_isShownBy_attributes][media]',
+            type:       'file'
```

Comment:
Could we either:
1. change this to `is_file: true`; or
2. change `is_select: true` to `type: 'select'` and `is_textarea: true` to `type: 'textarea'`

Reply:
The conventions I added follow the structure of html elements: the "is_" prefix is used on the two that use diiferent tags (select and textarea).  All other types of html input (text, date, file, hidden) are differentiated only by a "type" attribute.  I thought this made for a cleaner mapping, though as you point out they do appear inconsistent when listed in the ruby file.